client/extensions/woocommerce/state/sites/locations/selectors.js

```diff
@@ -72,6 +72,23 @@ export const getCountries = ( state, continentCode, siteId = getSelectedSiteId(
  * @param {Object} state Whole Redux state tree
  * @param {String} countryCode 2-letter ISO country code
  * @param {Number} [siteId] Site ID to check. If not provided, the Site ID selected in the UI will be used
+ * @return {String} The country name.
```

Comment:
Maybe mention that the countryCode will sometimes be returned if a country name can't be found/loaded.